### ISSUE TITLE
Update native crypto / ERC 20 token screens

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#1995]
Do we have a refreshed design for this screen which includes how the fiat value is displayed similar to https://app.zeplin.io/project/5d0356be2f25b319dcc5944b/screen/5ea921d764c254240f67a8b5 ?

<img src="https://user-images.githubusercontent.com/56189/86433406-45c5f280-bd2d-11ea-9e5c-b4d475ee9f72.png" width=200>

We'll need a few variations like in the wallet tab —

* native crypto where fiat value is available
* native crypto where fiat value is not available
* tokens, not fiat value
* native crypto or tokens for testnets

**Comment by tomekalphawallet** [comment]
- native crypto where fiat value is not available

What is an example of this?

**Comment by hboon** [comment]
Like ETC and Ropsten ETH:

<img src="https://user-images.githubusercontent.com/56189/86441212-6cd8f000-bd3e-11ea-9724-3d484480fd7b.png" width=200>

**Comment by tomekalphawallet** [comment]
- **native crypto where fiat value is available**

You can tap to switch the second line. 
By default we show: **Value $1359.43 USD**. 
On tap we switch to **Market Price: $226.79 USD (+2.57%)** 
And on you can tap once again to show **Available: 4.244 ETH**
<img width="1833" alt="Screenshot 2020-07-03 09 40 24" src="https://user-images.githubusercontent.com/51817359/86444675-67b17c00-bd11-11ea-8842-b7003086cc57.png">

**Comment by hboon** [comment]
@tomekalphawallet what if:

1. What if there is no fiat value? Do we just hide the second line?
2. How do we indicate testnets?

**Comment by tomekalphawallet** [comment]
<img width="1480" alt="Screenshot 2020-07-03 09 41 24" src="https://user-images.githubusercontent.com/51817359/86445234-4ef59600-bd12-11ea-8362-55e600351318.png">

- **native crypto where fiat value is not available**
When a fiat value is not available, we show **Available: 4.3455 ETH**
Available balance is what was left after a transaction, that was not confirmed yet. It might not be relevant with ETH 2.0, where transactions are much faster. 

- **tokens, not fiat value**
Keep in mind, that Activity section start in the same moment as with Ether + ERC 20 tokens. 
<img width="1019" alt="Screenshot 2020-07-03 09 55 14" src="https://user-images.githubusercontent.com/51817359/86445809-549fab80-bd13-11ea-9828-c3bef5a3bd04.png">

- **native crypto or tokens for testnets**
As in the screenshot

We also have another type of token: DEFI. On tap you switch between **Interest Rate: 5.75%** and **Earned so far: 4.3343 ETH**

- **DEFI token with interest rate and earned value**

<img width="1083" alt="Screenshot 2020-07-03 09 59 42" src="https://user-images.githubusercontent.com/51817359/86446337-1bb40680-bd14-11ea-8ade-300627900e4f.png">

**Comment by hboon** [comment]
@tomekalphawallet Great. Thanks. Is there Zeplins to pick out the fonts and colors? Especially for the leftmost screen in this:

<img width="1480" alt="86445234-4ef59600-bd12-11ea-8362-55e600351318" src="https://user-images.githubusercontent.com/56189/86447392-df020280-bd47-11ea-8d92-1636e0734e8b.png">

**Comment by tomekalphawallet** [comment]
https://app.zeplin.io/project/5d0356be2f25b319dcc5944b/styleguide/components?cseid=5efeeb8d3e258fb78e39e09b

**Comment by colourful-land** [comment]
@hboon if we make a token card for this (so that you don't have to implement this in SWIFT - neither James Brown) for shipping with the product (as a special TokenScript file that has no "contract"), would it work for you?

If so, I'll move this issue to the Tokenscript project so work is being done there.

**Comment by hboon** [comment]
@colourful-land depends on timeline and reason for doing it. For the short term, it's actually more difficult to display a new type of token card here. Updating the UI in native code is already worked on at #2014.

**Comment by colourful-land** [comment]
> @colourful-land depends on timeline and reason for doing it. For the short term, it's actually more difficult to display a new type of token card here. Updating the UI in native code is in progress at #2014.

{Be aware you need to do this for all compound tokens, not one}

In that case, here I return what @tomekalphawallet posted in this issue:
> Compond Tokens:

> On tap you switch between **Interest Rate: 5.75%** and **Earned so far: 4.3343 ETH**

> - **DEFI token with interest rate and earned value**

> <img width="1083" alt="Screenshot 2020-07-03 09 59 42" src="https://user-images.githubusercontent.com/51817359/86446337-1bb40680-bd14-11ea-8ade-300627900e4f.png">

The issue in TokenScript Examples remains

https://github.com/AlphaWallet/TokenScript-Examples/issues/99

**Comment by hboon** [comment]
Yes, and I think we can still use the TokenScript file to define the attributes for earnings and interest rate and code the calculation and formatting (the JavaScript part) in Swift.
